### PR TITLE
Revise less equal

### PR DESCRIPTION
**Pull request by bszmelcz**
### Details:
 - update spec
 - add backend test
 - serialize SLT is added as a part of the [PR](https://github.com/openvinotoolkit/openvino/pull/6605)
 - visitors test is added as a part of the [PR](https://github.com/openvinotoolkit/openvino/pull/6667)

### Tickets:
 - 37446

### CODE REVIEW

File: docs/ops/comparison/LessEqual_1.md [hunk 1]
```diff
@@ -4,48 +4,50 @@
 
 **Category**: Comparison binary operation
 
-**Short description**: *LessEqual* performs element-wise comparison operation with two given tensors applying multi-directional broadcast rules.
+**Short description**: *LessEqual* performs element-wise comparison operation with two given tensors applying multi-directional broadcast rules specified in the *auto_broadcast* attribute     .
```
Comment (sdurawa) [inline docs/ops/comparison/LessEqual_1.md:7]: remove blank spaces before ending dot.

File: docs/template_plugin/tests/functional/op_reference/less_eq.cpp [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+}
+
+std::vector<LessEqualParams> generateLessEqualCombinedParams() {
+    const std::vector<std::vector<LessEqualParams>> LessEqualTypeParams {generateLessEqualParams<element::Type_t::f32>(ngraph::element::f32),
+                                                                         generateLessEqualParams<element::Type_t::f16>(ngraph::element::f16),
+                                                                         generateLessEqualParams<element::Type_t::i32>(ngraph::element::i32),
+                                                                         generateLessEqualParams<element::Type_t::u32>(ngraph::element::u32),
+                                                                         generateLessEqualParams<element::Type_t::u8>(ngraph::element::boolean)};
+    std::vector<LessEqualParams> combinedParams;
+    std::for_each(LessEqualTypeParams.begin(), LessEqualTypeParams.end(), [&](std::vector<LessEqualParams> params) {
```
Comment (sdurawa) [inline docs/template_plugin/tests/functional/op_reference/less_eq.cpp:105]: switch to range-base loop

**Review by sdurawa — COMMENTED** [review]
Remove LessEqual tests from backend tests.

**Review by ilyachur — COMMENTED** [review]
We have the same activity in the PR: https://github.com/openvinotoolkit/openvino/pull/6728

File: docs/ops/comparison/LessEqual_1.md [hunk 3]
```diff
@@ -4,48 +4,50 @@
 
 **Category**: Comparison binary operation
 
-**Short description**: *LessEqual* performs element-wise comparison operation with two given tensors applying multi-directional broadcast rules.
+**Short description**: *LessEqual* performs element-wise comparison operation with two given tensors applying multi-directional broadcast rules specified in the *auto_broadcast* attribute.
```
Comment (lazarevevgeny) [inline docs/ops/comparison/LessEqual_1.md:7]: remove word "multi-directional"

**Review by lazarevevgeny — APPROVED** [review]
Minor comment for the spec.

**Review by ilyachur — APPROVED** [review]

**Comment by ilyachur** [comment]
Please fix CI

**Pull request merged**